launcher_go/v2/config/config_test.go

```diff
@@ -0,0 +1,39 @@
+package config_test
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/discourse/discourse_docker/launcher_go/v2/config"
+	"os"
+	"strings"
+)
+
+var _ = Describe("Config", func() {
+	var testDir string
+	var conf *config.Config
+	BeforeEach(func() {
+		testDir, _ = os.MkdirTemp("", "ddocker-test")
+		conf, _ = config.LoadConfig("../test/containers", "test", true, "../test")
+	})
+	AfterEach(func() {
+		os.RemoveAll(testDir)
+	})
+	It("should be able to load", func() {
+		conf, err := config.LoadConfig("../test/containers", "test", true, "../test")
+		Expect(err).To(BeNil())
+		result := conf.Yaml()
+		Expect(string(result)).To(ContainSubstring("DISCOURSE_DEVELOPER_EMAILS: 'me@example.com,you@example.com'"))
+		Expect(string(result)).To(ContainSubstring("_FILE_SEPERATOR_"))
+		Expect(string(result)).To(ContainSubstring("version: tests-passed"))
```

Comment:
Since we are dealing with the `Config` struct here, can't we assert against the fields instead of converting it to a string and doing partial match?

Reply:
yaml at this point is the concatenated string of all config in yaml form. We didn't need `string(result)` though, so I've updated that.